templates/includes/head.html

```diff
@@ -48,7 +48,7 @@
 <!-- Scripts
 ================================================== -->
 <script src="/static/js/vendor/modernizr-2.8.3.min.js"></script>
-<script src="/static/js/vendor/jquery.min.js"></script>
+<script src="/static/js/vendor/jquery-3.2.1.min.js"></script>
```

Comment:
I updated jQuery to 3.2.1, and put the version number explicitly in the filename.